spec/core.md

```diff
@@ -18,187 +18,253 @@ DID, after the prefix, is the case-sensitive [[ref: method-specific identifier]]
+current private key is compromised.
 
 A target system cannot forge or tamper with data protected by KERI, and if it
 deliberately serves an outdated copy, the duplicity is often detectable. Thus,
 any given target system in isolation can be viewed by this method as a dumb,
-untrusted server of content. It is the combination of target systems and a few
+untrusted server of content. It is the combination of target systems and some
 KERI mechanisms, _together_, that constitutes this method's verifiable data
-registry.
+registry. In short, verifying the DIDDoc by processing the KEL using KERI puts
+the "s" in `did:webs`.
 
 ### Method-Specific Identifier
 
-There are two forms of [[ref: method-specific identifier]] ([[ref: MSI]]) for
-`did:webs`. One is _resolvable_; the other is _stable_. The resolvable form is
-more common and intuitive.
-
-### Resolvable Form
-
-A [[ref: resolvable]] MSI is constructed by transforming an input called the
-[[ref: DID doc URL]] ([[ref: DDURL]]). This MUST be an HTTPS URL that references
-a DID doc (either directly, or ultimately, via redirects). The transformation to
-MSI must be reversible back to DDURL, because resolving the DID requires clients
-to fetch DID docs from the MSI.
-
-The DDURL MUST NOT include a query string (although it could redirect to a URL
-that contains one). The DDURL MUST NOT include relative path components ("./" or
-"../"). The DDURL MUST NOT include multiple consecutive slashes once the path
-portion begins. The DDURL MUST NOT include a fragment OR redirect to a URL that
-includes a fragment. It MUST return a resource that the web server identifies in
-headers as IANA media type `application/json`. The DDURL and its corresponding
-X509 certificate(s) also have some security requirements described
-[elsewhere](#security_considerations).
-
-The transformation begins by removing the "https://" prefix from this URL. If
-the path portion of the URL consists exactly of "/.well-known/did.json", this
-suffix is also removed. If the URL ends in a slash, the final slash is removed.
-All remaining slashes in the URL are replaced with colons. If the URL ends with
-".json", this suffix is deleted. If the URL includes a port, the colon that
-delimits the port number MUST be percent-encoded so colons in the MSI map only
-to slashes.
-
+The `did:webs` method specific identifier should be considered to have two
+parts, a fully qualified domain name with an optional path (identical to
+`did:web`), plus a KERI AID (autonomic identifier) that is always the final
+component of the path. A KERI AID is a unique-identifier derived from the
+inception event of a KERI identifier--the first item in the [[ref: KEL]]. Any
+KERI identifier (DIDs or AIDs) that reference the same AID must, by definition,
+refer to the same KEL, and hence, be the same identifier. An `did:webs` DIDs
+that have the same AID are by definition, synonyms of one another.
+
```

Comment:
"Any KERI identifier (DIDs or AIDs) that reference the same AID..." I think this wording will be confusing to different audiences for different reasons. KERI purists will say that KERI doesn't have DIDs. People who don't know KERI won't understand what AIDs are. So to simplify, perhaps just say, "Any identifier that references the same AID must, by definition, refer to the same KEL...". Also, I think "hence, be the same identifier" should be, "hence, identify the same subject." They are NOT the same identifier; it is what they reference that's the same.

Note the missing "y" at the and of "An" in the final sentence of this paragraph.

Reply:
Fixed